client/src/pages/SearchResultsUser.js

```diff
@@ -1,23 +1,53 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
 import { Typography } from "@material-ui/core";
-import SearchList from "../Components/SearchResList";
+import SearchResList from "../Components/SearchResList";
 import { useLocation } from "react-router-dom";
 
 function SearchResultsUser() {
 
-// Will get the query from the URL: https://codesandbox.io/s/react-router-query-parameters-mfh8p?from-embed
-  const useQuery = () => {
+const [results, setResults] = useState(null);  
+const [error, setError] = useState(null);
+const useQuery = () => {
     return new URLSearchParams(useLocation().search);
   }
 
-  const query = useQuery().get("query");
+//"https://codesandbox.io/s/react-router-query-parameters-mfh8p?from-embed"
+  useEffect(()=>{
+      fetch(events)
+        .then(response => {
+          if (!response.ok) {
+            setError ("Sorry something went wrong");
+            return;
+          } return response.json();
+        }).then(events => {
+          setState(events);
```

Comment:
`setState` won't work here, because that's not what you called the state setting method for results. You need to call the `setResults` method that you defined up on Line #8 along with the `results` state.

`setState` is the old method that `componentDidMount` used to use where you gave it an object of all the states at once every time you had to change it.